README.md

```diff
@@ -94,12 +94,23 @@ Install **both** of the following:
    - `Functions-request-config.js` which contains the `request` object that has all the data necessary to trigger a Functions request. This config file also specifies which `source` code to pass to Functions. More information on request configuration is in the [Request Configuration section](#request-configuration).
    - `contracts/FunctionsConsumer.sol` is the consumer smart contract that will receive the Functions-related data from the request config, and trigger the functions request.
    - `calculation-example.js` contains JavaScript code that will be executed by each node of the DON. This example performs complex calculations but no API requests.
-   - `API-request-example.js` which fetches data from APIs before processing the data <br><br>
-6. Simulate the JavaScript code for the request locally to ensure it produces the expected output:<br>`npx hardhat functions-simulate-script`<br><br>
+   - `./API-request-example.js` which fetches data from APIs before processing the data <br><br>
+6. Locally simulate the execution of your JavaScript source by using:<br>`npx hardhat functions-simulate-script`.
+
 7. Deploy and verify the consumer contract to an actual blockchain network by running:<br>`npx hardhat functions-deploy-consumer --network network_name_here --verify true`<br>**Note**: Make sure `<explorer>_API_KEY` is set if using `--verify true`, depending on which network is used.<br><br>
-8. Create and fund a new Functions billing subscription. Make a note of your subscription Id as you will need it for most commands. Then authorize your deployed consumer contract to use that subscription by running:<br> `npx hardhat functions-sub-create --network network_name_here --amount LINK_funding_amount_here --contract 0xDeployed_consumer_contract_address_here`
-   <br>**Note**: Ensure your wallet has a sufficient LINK balance before running this command. Testnet LINK can be obtained at <a href="https://faucets.chain.link/">faucets.chain.link</a>.<br><br>
-9. Make an on-chain request by running:<br>`npx hardhat functions-request --network network_name_here --contract 0xDeployed_consumer_contract_address_here --subid subscription_id_number_here`
+8. Create and fund a new Functions billing subscription, and then authorize your deployed consumer contract to use that subscription by running:<br> `npx hardhat functions-sub-create --network network_name_here --amount LINK_funding_amount_here --contract 0xDeployed_client_contract_address_here`<br><br>**Note**: Ensure your wallet has a sufficient LINK balance before running this command. Testnet LINK can be obtained at <a href="https://faucets.chain.link/">faucets.chain.link</a>.
```

Comment:
I am wondering if, for the quickstart, we should just instruct users on using the frontend to create their subscription as they will already need to use the frontend to accept the terms of service/add themselves to the allowlist.
I think the majority of users will just use the frontend for subscription management anyway.

Reply:
I don't think the instructions need to be detailed since this step is so straightforward. Maybe just say something like "create a subscription via the Chainlink Functions website" & give them a link.

Reply:
Make sure to use "consumer", not "client"